.env.example

```diff
@@ -1,14 +1,30 @@
 # Copy this file to .env and fill in the values
-STARKNET_RPC_URL=http://0.0.0.0:5050/rpc
+
+# Rust Environment
 RUST_LOG=debug
+
+# Starknet Environment
+STARKNET_NETWORK=testnet
+### Starknet provider urls:
+# {
```

Comment:
This is already hardcoded in starknet rs, see https://github.com/xJonathanLEI/starknet-rs/blob/35c287e1a06e6ab68447f5f0b9df53f910960f57/starknet-providers/src/sequencer/mod.rs#L114-L136